data/tabular/skin_reaction/meta.yaml

```
@@ -59,3 +61,86 @@ bibtex:
       title = {Predicting chemically-induced skin reactions. Part I: QSAR models of skin sensitization
       and their application to identify potentially hazardous compounds},
       journal = {Toxicology and Applied Pharmacology}
+templates:
+    - The {#molecule |!}{SMILES__description} {SMILES#} causes {skin_reaction#no &NULL}reaction with the skin.
```

Why not using the nouns?
```suggestion
    - The {#molecule |!}{SMILES__description} {SMILES#} causes {skin_reaction#no &NULL}{skin_reaction__names__noun}.
```